packages/compiler/src/core/parser.ts

```diff
@@ -269,7 +267,6 @@ namespace ListKind {
     delimiter: Token.Comma,
     toleratedDelimiter: Token.Semicolon,
     toleratedDelimiterIsValid: false,
-    trailingDelimiterIsValid: false,
     invalidAnnotationTarget: "expression",
```

Comment:
There is one case where it still doesn't work and its the heritage case. 
```
interface A {}; 
interface B {};
interface O extends A, B, {} // Bad diagnostic, which is curious
```
 that was already not showing the `trailing-token` error before either and this really also feels pointless to really support as the goal of trailing delimiter is usally to reduce diff when adding a new line on multi line and for interface interitance the last line is already trailed with `{`